lib/house.rb

```diff
@@ -0,0 +1,62 @@
+
+require './lib/room'
+
+class House
+  attr_reader :price,
+              :address
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @area = area
+  end
+
+  def rooms
+    @rooms
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
```

Comment:
just a nitpick but there is no need for the `if else` statement here as `@price.scan(/[0-9]/).join.to_i > 500000` resolves to a boolean